plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreAuthenticationModule.java

```diff
@@ -70,7 +71,10 @@ public HadoopAuthentication createHadoopAuthentication(MetastoreKerberosConfig c
         {
             String principal = config.getHiveMetastoreClientPrincipal();
             String keytabLocation = config.getHiveMetastoreClientKeytab();
-            return createCachingKerberosHadoopAuthentication(principal, keytabLocation, updater);
+            KerberosConfiguration.Builder builder = new KerberosConfiguration.Builder()
+                    .withKerberosPrincipal(principal)
+                    .withKeytabLocation(keytabLocation);
+            return createCachingKerberosHadoopAuthentication(builder.build(), updater);
```

Comment:
nit: call the `.build()` method above, so you can declare `KerberosConfiguration configuration`. instead `KerberosConfiguration.Builder builder`